CHANGELOG/CHANGELOG-4.0.md

```diff
@@ -0,0 +1,261 @@
+# Release notes for v4.0.0
+
+[Documentation](https://kubernetes-csi.github.io)
+
+# Changelog since v3.0.0
+
+## Breaking Changes
+
+- Update to client/v4 ([#451](https://github.com/kubernetes-csi/external-snapshotter/pull/451), [@xing-yang](https://github.com/xing-yang))
+
+- Move Snapshot APIs to V1. ([#419](https://github.com/kubernetes-csi/external-snapshotter/pull/419), [@xing-yang](https://github.com/xing-yang))
```

Comment:
Adding the v1 API is not breaking because we're still keeping v1beta right?

Reply:
Right. It's only the controller changes which requires this API change.  Let me try to clarify that.